### ISSUE TITLE
Improve description of themes

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons-frontend#926]
One of the things that came up in the accessibility review was that the themes did not provide enough contextual information for a screenreader user to understand what it was.

I wonder if we should make either make a change adding more of an explanation for all users or just some additional context just for screenreaders. 

Ideally I think the former makes sense because if you look at the themes and don't interact with it and see what it does, it's a bit of a mystery - we should try not to rely solely on visuals to infer information - that way it helps users with varying needs more.

@designakt @pwalm I'd be interested on your input on this especially if this ties into updates related to the user research.

**Comment by muffinresearch** [comment]
Relates to https://github.com/mozilla/addons/issues/179

**Comment by designakt** [comment]
The theme images alt text currently states: "Hover to preview or click to install Tender Spring Rain"
Let's change that to "Hover to preview or click to install **this theme:** Tender Spring Rain"

**Comment by muffinresearch** [comment]
Theme previews are going away: https://github.com/mozilla/addons-frontend/issues/3697

**Comment by kumar303** [comment]
> Theme previews are going away: #3697

Did you link to the right issue? That one is about a license view.

**Comment by designakt** [comment]
I think he is talking about this: #3056
Sad to hear that previews go away. They made themes so much more relatable.

**Comment by tofumatt** [comment]
FWIW: Currently a theme has to be previewed by tapping a button, which is the same kind of interaction as tapping the install button/switch to preview/install then tap again to uninstall, so the UX should be pretty close.
